### PR TITLE
docs: update readme en/cn for jobs module features

### PR DESCRIPTION
**Pull request by MaikoTan**
This PR mainly do the following:

- update the jobs module screenshot and so the features of it
- update the jobs table for all the features currently
- better wording for the jobs module description
- fix several Chinese mistaken / poor translation
- fix Chinese characters spacing with Latin characters following [pangu.js](https://github.com/vinta/pangu.js)

### CODE REVIEW

**Comment by Echoring** [comment]
Update description of jobs module. Also fix some pool translation.
I screenshot a new image of jobs module, for the image used now is too old and everything has changed.
Can anyone check is this image size appropriate?

File: README.md [hunk 1]
```diff
@@ -346,47 +346,54 @@ You can copy oopsy lines to the clipboard by clicking them.
...
-|<img src="./resources/ffxiv/jobs/blm-large.png" width="30px"/><br> Black Mage|Shows DoTs remaining time, Firestarter and Thundercloud proc duration, time to next xeno, MP ticker, Fire/Ice stack and umbral heart stack.|
-|<img src="./resources/ffxiv/jobs/smn-large.png" width="30px"/><br> Summoner|Shows Demi-Summoning time, Aetherflow stack, Attunement stacks, Energy Drain cooldown, Trance cooldown, and Lucid Dreaming cooldown.|
-|<img src="./resources/ffxiv/jobs/rdm-large.png" width="30px"/><br> Red Mage|Shows white/black mana, tracks procs for Verstone and Verfire, and Fleche and Contre Sixte cooldown.|
-|<img src="./resources/ffxiv/jobs/blu-large.png" width="30px"/><br> Blue Mage|Shows cooldown of offguard, lucid dreaming, and Song Of Torment DoT remaining time.|
+You can change some of the behavior or appearence via the user panel, e.g. only show the raidbuff zone, or enable compact view.
 
-</details>
+However, customization of some behavior like cooldown alert thresholds and element order is not availble for now.
```
Comment (quisquous) [inline README.md:357]:
```suggestion
However, customization of some behavior like cooldown alert thresholds and element order is not available for now.
```

File: README.md [hunk 2]
```diff
@@ -346,47 +346,54 @@ You can copy oopsy lines to the clipboard by clicking them.
...
 
-In this screenshot, the jobs module is highlighted for the Red Mage job. The health and mana bars, as well
-as Red Mage white/black mana tracking is circled in purple, with the large raid buff tracking pointed to
-beside it in orange. <del>The first step of the melee combo has been executed, which is displayed as the yellow
-box above the health bar.</del> The proc tracking is circled below in green.
+In this screenshot, RDM jobs UI is shown as example.
```
Comment (quisquous) [inline README.md:359]:
```suggestion
In this screenshot, the RDM jobs UI is shown as an example.
```

File: README.md [hunk 3]
```diff
@@ -346,47 +346,54 @@ You can copy oopsy lines to the clipboard by clicking them.
...
+The ingame UI is shown at top and jobs module is shown at bottom.
```
Comment (quisquous) [inline README.md:360]:
```suggestion
The in-game UI is shown at top and the jobs module is shown at bottom.
```

**Review by quisquous — COMMENTED** [review]
Thanks for the new images, that's nice!

File: docs/zh-CN/README.md [hunk 4]
```diff
@@ -188,217 +195,229 @@ ThirdParty
...
-cactbot已在当前版本的国际服（英语、德语、法语、日语）、国服以及韩服经过测试并可以正常工作。 某些翻译工作仍在进行中。
+cactbot 已在当前版本的国际服（英语、德语、法语、日语）、国服以及韩服经过测试并可以正常工作。某些翻译工作仍在进行中。
 
 ## 许可、商标与授权
 
-cactbot基于[Apache License, Version 2.0](../../LICENSE)开放源代码。
+cactbot 基于 [Apache License, Version 2.0](../../LICENSE) 开放源代码。
 
-FINAL FANTASY是株式会社史克威尔艾尼克斯控股(株式会社スクウェア・エニックス・ホールディングス，Square Enix Holdings Co., Ltd.)的注册商标。
+FINAL FANTASY 是株式会社史克威尔艾尼克斯控股 (株式会社スクウェア・エニックス・ホールディングス，Square Enix Holdings Co., Ltd.) 的注册商标。
```
Comment (MaikoTan) [inline docs/zh-CN/README.md:419]: Should use the official localized registered name for referring Square Enix

```suggestion
FINAL FANTASY 是史克威尔艾尼克斯控股公司 (株式会社スクウェア・エニックス・ホールディングス，Square Enix Holdings Co., Ltd.) 的注册商标。
```

**Comment by MaikoTan** [comment]
OK, it is good to merge now!

**Comment by quisquous** [comment]
Thank you!!

**Pull request merged**